File: Samples/Verbum/richeditcontrol-event-viewer-t258534-18.2.5-/CS/RichEditControlEventViewer/AutoCorrectEventHooker.cs
using System;
using System.Collections.Generic;
using System.Linq;

namespace RichEditControlEventViewer
{
    #region RichEditEventHooker
    public class AutoCorrectEventHooker : RichEditEventHooker
    {
        public AutoCorrectEventHooker(string eventName, Form1 owner) :
            base(eventName, owner)
        {
        }
        protected override string MethodName { get { return "InitializeAutoCorrectEventLogger"; } }
    }
    #endregion
}

File: Samples/Verbum/richeditcontrol-event-viewer-t258534-18.2.5-/VB/RichEditControlEventViewer/AutoCorrectEventHooker.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq

Namespace RichEditControlEventViewer
    #Region "RichEditEventHooker"
    Public Class AutoCorrectEventHooker
        Inherits RichEditEventHooker

        Public Sub New(ByVal eventName As String, ByVal owner As Form1)
            MyBase.New(eventName, owner)
        End Sub
        Protected Overrides ReadOnly Property MethodName() As String
            Get
                Return "InitializeAutoCorrectEventLogger"
            End Get
        End Property
    End Class
    #End Region
End Namespace
